asab/docker/__init__.py

```diff
@@ -14,11 +14,10 @@
 
 Config.add_defaults(
 	{
-		'asab:docker': {
+		'docker': {
 			# Docker API or socket
 			# Could be `http://myHost:2375` or `/var/run/docker.sock`
-			'socket': '',
-			'name_prefix': '',
+			'socket': '/var/run/docker.sock',
```

Comment:
Will this nicely work if the asab microservice is not running in the docker (1) or in a docker without configured path (2)?

Reply:
(1) yes, I think. (2) I have no idea. I'm giving up. I don't like default configs anyway.